packages/common/src/operations.graphql

```diff
@@ -402,152 +395,154 @@ mutation insertStocktake($input: InsertStocktakeInput!) {
+#   batchSupplierRequisition(deleteSupplierRequisitions: $ids) {
+#     __typename
+#     deleteSupplierRequisitions {
+#       __typename
+#       id
+#     }
+#   }
+# }
 
-mutation updateSupplierRequisition($input: UpdateSupplierRequisitionInput!) {
-  updateSupplierRequisition(input: $input) {
+mutation updateSupplierRequisition(
+  $storeId: String!
+  $input: UpdateRequestRequisitionInput!
+) {
+  updateRequestRequisition(input: $input, storeId: $storeId) {
     ... on RequisitionNode {
       __typename
       id
     }
   }
 }
 
-mutation insertSupplierRequisition($input: InsertSupplierRequisitionInput!) {
-  insertSupplierRequisition(input: $input) {
+mutation insertSupplierRequisition(
+  $storeId: String!
+  $input: InsertRequestRequisitionInput!
+) {
+  insertRequestRequisition(input: $input, storeId: $storeId) {
     ... on RequisitionNode {
       __typename
       id
     }
   }
 }
 
-mutation deleteCustomerRequisitions($ids: [DeleteCustomerRequisitionInput!]) {
-  batchCustomerRequisition(deleteCustomerRequisitions: $ids) {
-    __typename
-    deleteCustomerRequisitions {
-      __typename
-      id
-    }
-  }
-}
+# mutation deleteCustomerRequisitions($ids: [DeleteRequestRequisitionInput!]) {
+#   batchRequestRequisition(deleteCustomerRequisitions: $ids) {
+#     __typename
+#     deleteRequestRequisitions {
+#       __typename
+#       id
+#     }
+#   }
+# }
 
-mutation updateCustomerRequisition($input: UpdateCustomerRequisitionInput!) {
-  updateCustomerRequisition(input: $input) {
+mutation updateCustomerRequisition(
+  $storeId: String!
+  $input: UpdateRequestRequisitionInput!
+) {
+  updateRequestRequisition(storeId: $storeId, input: $input) {
     ... on RequisitionNode {
       __typename
       id
     }
   }
 }
 
-mutation insertCustomerRequisition($input: InsertCustomerRequisitionInput!) {
-  insertCustomerRequisition(input: $input) {
+mutation insertCustomerRequisition(
+  $storeId: String!
+  $input: InsertRequestRequisitionInput!
+) {
+  insertRequestRequisition(input: $input, storeId: $storeId) {
     ... on RequisitionNode {
       __typename
       id
     }
   }
 }
 
-query requisition($id: String!) {
-  requisition(id: $id) {
+query requisition($storeId: String!, $id: String!) {
+  requisition(id: $id, storeId: $storeId) {
     __typename
     ... on RequisitionNode {
       __typename
       id
-      orderDate
-      requisitionDate
+      # orderDate
+      # requisitionDate
       comment
       theirReference
       type
       requisitionNumber
-      thresholdMOS
-      maxMOS
+      # thresholdMOS
+      # maxMOS
       status
       otherPartyId
       lines {
         __typename
-        ... on ConnectorError {
-          error {
-            description
-          }
-        }
         ... on RequisitionLineConnector {
           totalCount
           nodes {
             id
-            itemName
-            itemCode
-            itemUnit
+            # itemName
```

Comment:
Yeah.

I think we're really going to need to look at fragments now. I.e. a fragment `BasicItemNode`

```
fragment Item on ItemNode {
    id
    name
    code
  }
```

for example this will generate a new type - I believe `ItemFragment` is the default name - then I believe we should be able to make all of this a lot easier in some ways